docs/setuptools.txt

```diff
@@ -2293,6 +2293,44 @@ Metadata and options are set in the config sections of the same name.
 
 * Unknown keys are ignored.
 
+setup.cfg-only projects
+=======================
+
+.. note:: New in 40.9.0 (03 Apr 2019).
+
+Latest versions of ``setuptools`` emulate a dummy ``setup.py`` module
+containing only a ``setuptools.setup()`` call.
+
+This means that you can have a Python distribution project with no
```

Comment:
I'm not sure this paragraph is actually accurate. The critical point is that builds invoked with `setuptools.build_meta` do not require a `setup.py` file and can pull all configuration from `setup.cfg`. In practice, the way to invoke a build with `setuptools.build_meta` is to add a `pyproject.toml` file and use a PEP 517-capable build-frontend like `pip`, `tox` or `pep517.build`.

Reply:
So how do you want me to change this?